www/include/monitoring/downtime/AddDowntime.php

```diff
@@ -315,11 +317,19 @@
 
     $data = array();
     $gmt = $centreonGMT->getMyGMT();
-    if(!$gmt) {
+    if (!$gmt) {
         $gmt = date_default_timezone_get();
     }
+    if ($defaultScale == 'm') {
+        $defaultDuration *= 60;
+    } elseif ($defaultScale == 'h') {
+        $defaultDuration *= 3600;
+    } elseif ($defaultScale == 'd') {
+        $defaultDuration *= 86400;
+    }
     $data["start_time"] = $centreonGMT->getDate("G:i", time(), $gmt);
-    $data["end_time"] = $centreonGMT->getDate("G:i", time() + 7200, $gmt);
+    $data["end_time"] = $centreonGMT->getDate("G:i", time() + $defaultDuration, $gmt);
+    $data["end"] = $centreonGMT->getDate("m/d/Y", time() + $defaultDuration, $gmt);
```

Comment:
Depending on the user local, this will not work (test for example with fr locale).
This is why I had to workaround in #7357.